src/aashay/Shop.java

```diff
@@ -0,0 +1,94 @@
+package aashay;
+
+/*
+ * 1. Create a system (Shop) which maintains each item's quantity.
+
+a. Initial stocks of Maggie: 50 Maggie packets
+b. Initial stocks of Dosa : 43 packets
+c. Initial stocks of pouches : 39 oil packets
+d. Initial stocks of panipuri : 43 packets
+e. Initial stocks of masala�s : 73 packets.
+
+Expectation:
+1) Only one Customer came for shopping, he picked different items with different quantities, print "running out of stock" message if a particular item is sold out otherwise deduct requested quantity from total quantity. Partial selling is allowed.
+2) Create a method which shows only those item which is "Out of stock".
+3) Create a method which shows only those item which is "Available in stock" & respective quantities.
+ */
+
+public class Shop {
+	
+	int maggie = 50;
+	int dosa = 43;
+	int pouches = 39;
+	int panipur = 43;
+	int masala = 73;
+	
+	void shoppingCart(int magi, int dsa, int poch, int ppuri, int msla) {
+		System.out.println("Today morning stock is:");
+		System.out.println("Maggie: "+maggie);
+		System.out.println("Dosa: "+dosa);
+		System.out.println("Pouches: "+pouches);
+		System.out.println("Panipuri: "+panipur);
+		System.out.println("Masala: "+masala);
+		
+		maggie = maggie - magi;
```

Comment:
what if requested quantity is more than available ?
need to handle this scenario.